### PR TITLE
README: Switch from trac & email patches to GitHub PRs & issues

### PR DESCRIPTION
**Pull request by henrik242**
Ref. http://lists.subsurface-divelog.org/pipermail/subsurface/2016-December/026860.html

Signed-off-by: Henrik Brautaset Aronsen <subsurface@henrik.synth.no>

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -75,12 +76,13 @@ There is a mailing list for developers: subsurface@subsurface-divelog.org
...
 
-If you want to contribute code, please either send signed-off patches or
-a pull request with signed-off commits.  If you don't sign off on them,
-we will not accept them. This means adding a line that says
-"Signed-off-by: Name <email>" at the end of each commit, indicating that
-you wrote the code and have the right to pass it on as an open source
-patch.
+If you want to contribute code, please open a pull request with signed-off
+commits at https://github.com/Subsurface-divelog/subsurface/pulls
+
```
Comment (Subsurface-divelog) [inline README]: I don't think I want to ONLY allow pull requests.
How about something like "The preferred way to contribute code is to open a pull request... but you can also send signed-off patches to our mailing list."
That seems like a gentler change (frankly, I don't know if all of our contributors are comfortable with github pull requests and I don't want to lose any more contributors...)

Comment (henrik242) [inline README]: Sure, no problem.  Shall I fix it our will you do it?

**Comment by dirkhh** [comment]
updated and merged

**Pull request closed without merging**